api/cotrip_app/migrations/0002_location.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.4 on 2020-03-30 18:46
+
+from django.db import migrations, models
+
+
```

Comment:
What's this doing here?

Reply:
Uhh.. I have no idea! I didn't think I did any work on the backend.. will try to exclude those